.github/CODEOWNERS

```diff
@@ -81,10 +82,11 @@
 
 
 # Interaction project.
-/core/templates/dev/head/domain/objects/ @kevinlee12
+# TODO(kevinlee12): Revert ownership to @kevinlee12 after 2019-04-10.
```

Comment:
Probably should specify the actual dirs, since this block has a mix of ownership.